storage/src/main/java/org/apache/uniffle/storage/handler/impl/MemoryClientReadHandler.java

```diff
@@ -65,15 +72,22 @@ public MemoryClientReadHandler(
       int partitionId,
       int readBufferSize,
       ShuffleServerClient shuffleServerClient,
-      Roaring64NavigableMap expectTaskIds,
-      boolean expectedTaskIdsBitmapFilterEnable) {
+      Roaring64NavigableMap expectBlockIds,
+      Roaring64NavigableMap processBlockIds,
+      boolean expectedTaskIdsBitmapFilterEnable,
+      IdHelper idHelper) {
     this.appId = appId;
     this.shuffleId = shuffleId;
     this.partitionId = partitionId;
     this.readBufferSize = readBufferSize;
     this.shuffleServerClient = shuffleServerClient;
-    this.expectTaskIds = expectTaskIds;
-    this.expectedTaskIdsBitmapFilterEnable = expectedTaskIdsBitmapFilterEnable;
+    if (expectedTaskIdsBitmapFilterEnable) {
```

Comment:
Could we calculate taskBitmap in the upper layer? Could we remove `expectBlockIds` and `processBlockIds`? It's enough to pass the paramter `expectTaskIds`?